cloudinit/sources/helpers/azure.py

```diff
@@ -194,7 +195,7 @@ def report_compressed_event(event_name, event_content):
         COMPRESSED_EVENT_TYPE, event_name,
         json.dumps(event_data),
         events.DEFAULT_EVENT_ORIGIN)
-    events.report_event(evt, excluded_handlers={"logging"})
+    events.report_event(evt, excluded_handler_types={"log"})
```

Comment:
@Moustafa-Moustafa I think we probably need to exclude {"log", "print", "webhook"} here otherwise we risk pushing that compressed data to the other log types too if configured
```suggestion
    events.report_event(evt, excluded_handler_types={"log", "print", "webhook"})
```

Reply:
Ultimately we don't need to push the log to KVP after upgrade and reboot. but I think it wouldn't be harmful in that case.